src/process/mod.rs

```diff
@@ -34,8 +34,10 @@ pub struct Argument {
 
 pub fn make_hl_ast(api_map: APIMap, conf: &conf::Generation) {
     debug!("Generation config: {:#?}", conf);
-    let functions = function_processor(&api_map, &conf.functions);
+    let functions = func::function_processor(&api_map, &conf.functions);
     debug!("Functions: {functions:#?}");
-    let namespaces = namespace_generator(&functions, &conf.namespaces);
+    let namespaces = namespace::namespace_generator(&functions, &conf.namespaces);
```

Comment:
The specification of the module is I think redundant, the function name is enough to know what is it